joomla/joomla-cms
# [4.0] Remove vueify - we use webpack in media manager

We don't actually use vueify - we use webpack to compile - it also causes a bunch of our registered `npm audit` issues and so this removes it.

## Testing Instructions
Try media manager and ensure there are no differences in behaviour from before patch

Also see media manager system tests pass
